service-loadbalancer/service_loadbalancer.go

```diff
@@ -110,6 +110,9 @@ var (
 
 	startSyslog = flags.Bool("syslog", false, `if set, it will start a syslog server
 		that will forward haproxy logs to stdout.`)
+
+	// content of the default index (to avoid read the file multiple times)
+	defaultIndex []byte
```

Comment:
instead of doing this can you make this a type:
```go
type staticPageHandler struct {
  pagePath string
  pageContents []byte
}

// load loads the page into memory 
func (s *staticPageHandler)load(){}

// Get serves the page stored by load
func (s *staticPageHandler)Get(http.ResponseWriter, *http.Request) {}

// newStaticPageHandler returns a staticPageHandles with the contents of pagePath loaded and ready to serve
func newStaticPageHandler(pagePath string) (*staticPageHandler, error) {}
```

That way it's more obvious what we're doing and we can reuse it to serve non default index pages as well.